crates/sui-types/src/messages.rs

```diff
@@ -1594,13 +1595,22 @@ impl Display for MoveLocation {
     fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
         let Self {
             module,
-            function,
+            function_idx,
             instruction,
+            message,
         } = self;
-        write!(
-            f,
-            "{module} in function definition {function} at offset {instruction}"
-        )
+        match message {
+            Some(s) => {
+                write!(f, "{s}")
+            }
+
```

Comment:
I think it is uncommon to have a newline here, just since you mentioned being a Rust noob :)

Reply:
ah cool, looks like default `rustfmt` is happy either way with this newline being here or not. I'll prefer deleting it thanks :-)